apps/ws/src/socket/ws.gateway.ts

```diff
@@ -199,4 +200,18 @@ export class WSGateway implements OnGatewayConnection, OnGatewayDisconnect, IDes
     this.isShutdown = true;
     await this.gracefulShutdown();
   }
+
+  afterInit() {
+    if (!!process.env.SOCKET_IO_ADMIN_USERNAME && !!process.env.SOCKET_IO_ADMIN_PASSWORD_HASH) {
+      instrument(this.server, {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Could we add a URL link to the configuration options for Socket.io? I find it useful to have a URL to click on to easily find the other options in case of maintenance + debugging. Specifically, I would like to know what effect the `mode` has on the admin panel.

Reply:
Do you mean as a comment?

Reply:
Perfect! 🚀